### PR TITLE
Add api docs

### PR DESCRIPTION
**Pull request by nickbristow**
# PULL REQUEST

## Summary
Creates docs for phdi ecr-viewer

## Related Issue
Fixes #

## Additional Information
Anything else the review team should know?

## Checklist

- [ ] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

**Review by BobanL — COMMENTED** [review]
just a couple of qs

File: containers/ecr-viewer/tsconfig.json [hunk 1]
```diff
@@ -30,6 +30,7 @@
     }
   },
   "include": [
+    "src/**/*",
```
[the remark this answers is not in the document]
Comment (nickbristow) [reply containers/ecr-viewer/tsconfig.json]: yep, leftover from troubleshooting

**Comment by nickbristow** [comment]
@BobanL i think this is good now

File: containers/ecr-viewer/package.json [hunk 2]
```diff
@@ -75,10 +76,14 @@
     "jest-environment-jsdom": "^29.7.0",
     "jest-extended": "^4.0.2",
     "jest-fetch-mock": "^3.0.3",
+    "jsdoc": "^4.0.3",
+    "jsdoc-plugin-typescript": "^2.2.1",
```
[the remark this answers is not in the document]
Comment (BobanL) [reply containers/ecr-viewer/package.json:80]: Wait so are both of these required still?

File: containers/tefca-viewer/package.json [hunk 3]
```diff
@@ -21,7 +21,7 @@
     "@aws-sdk/client-s3": "^3.515.0",
     "@trussworks/react-uswds": "^6.1.0",
     "@types/sinonjs__fake-timers": "^8.1.1",
-    "@uswds/uswds": "3.7.0",
+    "@uswds/uswds": "3.7.1",
```
Comment (BobanL) [inline containers/tefca-viewer/package.json:24]: You should probably revert this change in tefca-viewer.

File: containers/ecr-viewer/tsconfig.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (BobanL) [reply containers/ecr-viewer/tsconfig.json:33]: Should this still be here?

**Comment by nickbristow** [comment]
This is out of date now
